structured_prediction_baselines/modules/structured_score/sequence_tagging/self_attention.py

```diff
@@ -26,13 +45,15 @@ def forward(
     ) -> torch.Tensor:
         mask = buffer["mask"]
         batch_size, n_samples, seq_length, _ = y.shape
-        attention_mask = torch.BoolTensor(seq_length, seq_length).fill_(False)
-        for i in range(seq_length):
-            lower_idx, higher_idx = max(0, i - self.M), min(seq_length, i + self.M + 1)
-            attention_mask[i][lower_idx:higher_idx] = True
+        attention_mask = self._get_attention_mask(n_samples, mask)
+
+        if self._use_positional_encoding:
```

Comment:
Let's just reduce one line to
        attention_input = y.view(batch_size * n_samples, seq_length, -1) # change the shape to appropriate structure for self-attention function.
        if self._use_positional_encoding:
            attention_input = add_positional_features(attention_input)

Reply:
I directly committed actually reflecting the suggested changes above, does anybody know why I am not seeing the change in this PR?